backend/meal/serializers.py

```diff
@@ -51,6 +53,14 @@ class Meta:
         model = models.Meal
         fields = ("meal_ingredients",)
 
+    def validate(self, attrs):
+        """
+        Check that meal is not empty
+        """
+        if not attrs['meal_ingredients']:
+            raise serializers.ValidationError("Meal cannot be empty")
```

Comment:
What about changing the spelling, error is more like "Meal needs at least one ingredient"?

Reply:
OK